addon/components/bs-link-to.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable ember/classic-decorator-no-classic-methods */
+import Component from '@ember/component';
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { assert } from '@ember/debug';
+
+/**
+ This is largely copied from Ember.LinkComponent. It is used as extending from Ember.LinkComponent has been deprecated.
+ We need this to
+ * register ourselves to a parent component that needs to know `active` state due to Bootstrap markup requirements, see Nav/LinkTo
+ * continue supporting positional params until we can remove support
+
+ @class LinkComponent
+ @namespace Components
+ @extends Component
+ @private
+*/
+@tagName('')
+class LinkComponent extends Component {
+  // We still need to use the private service, so this component can react to changes of `currentState`, e.g. changing
+  // the model while the route in unchanged.
```

Comment:
@jelhan do you happen to know if there is another way to support this, using only public API?